requirements.txt

```diff
@@ -3,6 +3,7 @@ pandas
 pyparsing
 requests
 six
+tabulate
```

Comment:
you also need to add it in the setup.py
requirements.txt is just an info for the users of the package that are required but they are not installed when doing a pip install kili. The packages in the setup.py are installed when installing Kili